src/main/java/ch/njol/skript/tests/platform/Environment.java

```diff
@@ -220,14 +219,15 @@ public void initialize(Path dataRoot, Path runnerRoot, boolean remake) throws IO
 		}
 	}
 
-	public TestResults runTests(Path runnerRoot, Path testsRoot, boolean devMode, String... jvmArgs) throws IOException, InterruptedException {
+	public TestResults runTests(Path runnerRoot, Path testsRoot, boolean devMode, boolean genDocs, String... jvmArgs) throws IOException, InterruptedException {
 		Path env = runnerRoot.resolve(name);
 		List<String> args = new ArrayList<>();
 		args.add(System.getProperty("java.home") + File.separator + "bin" + File.separator + "java");
 		args.add("-ea");
 		args.add("-Dskript.testing.enabled=true");
 		args.add("-Dskript.testing.dir=" + testsRoot);
 		args.add("-Dskript.testing.devMode=" + devMode);
+		args.add("-Dskript.testing.genDocs=" + genDocs);
```

Comment:
Also add `skript.forceregisterhooks` to force generating hooks without the need to install the plugins

Reply:
Doesn't you pull request add this?

Reply:
It doesn't, it only checks if this property is used to generate the hooks as well regardless of whether the plugins are intalled or not